pkg/dialog/dialog.go

```diff
@@ -48,3 +52,13 @@ func MultiSelect(message string, items map[string]string) ([]string, error) {
 
 	return result, nil
 }
+
+func ReadSecret(message string, file *os.File) (string, error) {
+	log.Info(message)
+	log.Warnf("No characters will be displayed.")
```

Comment:
A warning seems a bit too much for this. Also, present tense seems more appropriate. How about this?

```suggestion
	log.Info("Note: Entered characters are not displayed")
```

Reply:
Also, I prefer the UX of `gh`, which prints `*` characters instead. It seems like none of the packages we currently import provide this functionality, so I suggest we add a `TODO` and keep the current behavior for now.

Reply:
Given that we are doing the right thing (not showing the characters), maybe we don't even have to print this message? The behavior is pretty common in CLI applications.

Reply:
Would work for me

Reply:
Removed the Warn/Info and added a Todo comment. I will also create a ticket.